MutationObserver.js

```diff
@@ -464,7 +501,7 @@ window.MutationObserver = window.MutationObserver || (function(undefined) {
                      */
                     elestruct.attr = reduce($target.attributes, function(memo, attr) {
                         if (!config.afilter || config.afilter[attr.name]) {
-                            memo[attr.name] = attr.value;
+                            memo[attr.name] = getAttributeValue($target, attr);
```

Comment:
Sorry, I did not get what you meant by *resolving a mutation*. From my understanding you cache `cssText` and then compare if it changed. I cannot think of any counterexamples now.